packages/steiger-plugin-fsd/src/no-wildcard-exports/README.md

```diff
@@ -0,0 +1,61 @@
+# `no-wildcard-exports`
+
+Forbid wildcard exports (`export *`) in public APIs of business logic layers. Named exports and namespace exports (`export * as namespace`) are allowed.
+
+**Exception:** Wildcard exports are allowed in unsliced layers (`shared` and `app`), as they serve as foundational layers with different architectural purposes.
```

Comment:
issue: I would actually argue against allowing wildcard exports even in Shared and App. To me, wildcard exports conceal the public API of a group of modules, which really hurts the ability to discover a codebase shallowly, i. e. without digging into every file. 

question: why do you think they should be allowed there? And do you agree with the content of the [Public API reference page in the FSD docs](https://feature-sliced.github.io/documentation/docs/reference/public-api)?